test/testapps/AppWithP2PReference/AppWithP2PReference.csproj

```diff
@@ -26,5 +27,19 @@
     <ProjectReference Include="..\ClassLibrary\ClassLibrary.csproj"/>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(BinariesRoot)'==''">
```

Comment:
Moved these from `Directory.Build.props` because we don't want these references tainting the RTM referencing class lib that was added